GitCommands/Git/ExecutableExtensions.cs

```diff
@@ -114,23 +110,21 @@ public static class ExecutableExtensions
                 redirectInput: input is not null,
                 redirectOutput: true,
                 outputEncoding,
-                throwOnErrorOutput: throwOnErrorOutput);
+                throwOnErrorOutput: true);
```

Comment:
This line can be removed, too. Or do you want to emphasize it?

Reply:
That was the intention, Can remove if you find it confusing.
(I have another argument to remove, then it is only visible when false.)

Reply:
I have no preference.